### ISSUE TITLE
Add support for 'as' and 'from' in import statement

### ISSUE THREAD

**Issue by flof** [PebbleTemplates/pebble#209]
It should be possible to use 'as' and 'from' when importing a macro file, like it's possible in [Twig](http://twig.sensiolabs.org/doc/tags/import.html) and [Jinja2](http://jinja.pocoo.org/docs/dev/templates/#import). This would mitigate name collisions and improve compatibility with Twig and Jinja2.

**Example with 'as' in import statement for setting an alias for the imported file:**

```
{% import 'forms.html' as forms %}
{{ forms.input('name') }}
```

**Example with 'from' in import statement for importing named macros:**

```
{% from 'forms.html' import input as input_text, textarea %}
{{ input_text('street') }}
{{ textarea('comment') }}
```

**Comment by lmajano** [comment]
This issue is almost 2 years old, will there be any development in the project again?

**Comment by tx-francesco-figari** [comment]
See #271

**Comment by ebussieres** [comment]
Resolved via #271
